src/index.html

```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="nav">
+        <div class="container">
+          <div class="logo"><img src="images/logo.png" alt="logo"></div>
```

Comment:
Logo should not be a part of `<nav>` according to readme Tips and Hints. remember that it will also require CSS changes.